engine/core/src/main/java/org/pentaho/reporting/engine/classic/core/modules/gui/html/HtmlStreamExportTask.java

```diff
@@ -73,7 +73,7 @@
   private File targetDirectory;
   private String suffix;
   private String filename;
-
+  private Boolean createParentFolder;
```

Comment:
Is there any particular reason you need an upper-case Boolean instead of the primitive "boolean"? Your boolean here will never be null according to your code, so why is this not a primitive?